recipes/mpfr/build.sh

```diff
@@ -0,0 +1,12 @@
+./configure --prefix=$PREFIX \
+            --with-gmp-include=$PREFIX/include \
+            --with-gmp-lib=$PREFIX/lib \
```

Comment:
Tried specifying using `--with-gmp`. However, there was a very nasty message about a mismatch between the include and library versions. So, trying this instead as this was suggested. Not expecting this will make a difference, but should rule it out anyways. Besides it always could work. :four_leaf_clover: